packages/extension-commerce-bm-seo/README.md

```diff
@@ -0,0 +1,108 @@
+:loudspeaker: Hey there, Salesforce Commerce Cloud community!
```

Comment:
Lets update this readme to be specific to this extension.

Reply:
We probably also want to note in the README that the extension has to come before other extensions so that the HOC works properly